install/hst-install-debian.sh

```diff
@@ -15,8 +15,8 @@
 #----------------------------------------------------------#
 export PATH=$PATH:/sbin
 export DEBIAN_FRONTEND=noninteractive
-RHOST='apt.hestiacp.com'
-GPG='gpg.hestiacp.com'
+# For testing
+RHOST='beta-apt.hestiacp.com'
```

Comment:
not sure if this is a good idea, i understand the idea behind, but maybe we should temporarely create a new file called hst-install-debian-beta.sh instead. Just to prevent that we don't change it back and mix productive with testing.